src/redux/books/books.js

```diff
@@ -1,50 +1,71 @@
-const initialState = [
-  {
-    id: 1,
-    title: 'Advance React',
-    author: 'Mosh',
-  },
-  {
-    id: 2,
-    title: 'Advance Redux',
-    author: 'Abdul Saboor',
+/* eslint-disable no-param-reassign */
```

Comment:
As much as possible, kindly avoid disabling linter checks in line. They represent coding standards you share with your team and most of the time enforce best coding practices.